pkg/kubelet/apis/config/types.go

```diff
@@ -346,6 +347,8 @@ const (
 	KubeletAuthorizationModeWebhook KubeletAuthorizationMode = "Webhook"
 )
 
+// KubeletAuthorization holds the state related to the authorization in the
```

Comment:
Nit: I'm not sure the second sentence, where you list what the struct includes, is entirely necessary. I worry it has the ability to grow out of date, should fields be added/removed from the struct. More important, in my mind, is capturing the purpose of the struct, and I believe your first sentence does that successfully.

Reply:
Thanks. I was wondering about this when I wrote it. I'll fix this up. 

What is standard way to fix this, add another commit or `--amend` and force push back to this branch so we only have a single commit?

Reply:
I have pushed a new commit to this PR that addresses these three comments. Should I "resolve" these conversations, or is that the responsibility of the original commenter?

Reply:
Either as fine - at some point before this diff is merged, I'd recommend squashing your commits into one (although the merge-robot tool may actually do so automatically when it merges the PR... I'm not positive).

Typically, I use `--amend`, unless I think my reviewer will want to look at the pr commit by commit.

Re resolving the comments, feel free to go ahead and resolve them :)

Reply:
Thanks, sorry for all the churn on this PR @mattjmcnaughton. I've squashed and rebased on `mater`.